hpat/tests/test_series.py

```diff
@@ -1556,6 +1557,32 @@ def test_impl(A):
         S = pd.Series(np.random.ranf(n))
         np.testing.assert_array_equal(hpat_func(S), test_impl(S))
 
+    def test_series_idxmin_str(self):
+        def test_impl(S):
+            return S.idxmin()
+        hpat_func = hpat.jit(test_impl)
+
+        S = pd.Series([8, 6, 34, np.nan], ['a', 'ab', 'abc', 'c'])
+        self.assertEqual(hpat_func(S), test_impl(S))
+
+    @unittest.skip("Enable after fixing index")
+    def test_series_idxmin_no(self):
+        def test_impl(S):
+            return S.idxmin()
+        hpat_func = hpat.jit(test_impl)
+
+        S = pd.Series([8, 6, 34, np.nan])
+        self.assertEqual(hpat_func(S), test_impl(S))
+
+    @unittest.skip("Enable after fixing index")
+    def test_series_idxmin_int(self):
+        def test_impl(S):
+            return S.idxmin()
+        hpat_func = hpat.jit(test_impl)
+
+        S = pd.Series([1, 2, 3], [4, 45, 14])
```

Comment:
please see this comment https://github.com/IntelPython/hpat/pull/217#discussion_r335113325

Reply:
Add 1 test and fix another